quiche/src/lib.rs

```diff
@@ -7050,7 +7120,7 @@ impl Connection {
 
     /// Processes an incoming frame.
     fn process_frame(
-        &mut self, frame: frame::Frame, hdr: &packet::Header,
+        &mut self, paths: &mut path::PathMap, frame: frame::Frame, hdr: &packet::Header,
```

Comment:
So how could we move process_frame to the path entirely, with some synchronization with the connection?